syntax = "proto3";

package grpc;

message Empty {}

message Mensaje {
    string message = 1;
}

service ChatServer {
    rpc passMessage (Mensaje) returns (Empty);
    rpc ControlMessage (Mensaje) returns (Mensaje);
    rpc AuthMessage(Mensaje) returns(Mensaje);
}
